src/fermo/app_utils/input_testing.py

```diff
@@ -0,0 +1,287 @@
+from werkzeug.utils import secure_filename
+import os
+import json
+
+# for type hinting
+from typing import Tuple
+from werkzeug.datastructures import FileStorage
+
+
+def check_file_format(
+    filename: str,
+    file_format: str,
+    allowed_extensions: list,
+) -> (str):
+    ''' Check filename for validity
+
+    Parameters
+    ----------
+    filename: `str`
+    file_format: `str`
+        File format of one specific input with the leading dot
+    allowed_extension: `list`
+        List of strings as specified in config file
+
+    Return
+    ------
+    feedback: `str`
+        String with feedback for the user to be flashed
+    '''
+    if '.' not in file_format:
```

Comment:
unit tests must not be in production code - please remove and create separate test cases

Reply:
upon further inspection this code turned out to be unnecessary and was removed